common/src/main/java/com/wynntils/features/stateless/LootrunFeature.java

```diff
@@ -23,7 +24,7 @@
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 @StartDisabled
-public class LootrunFeature extends InternalFeature {
```

Comment:
The LootrunFeature is a chapter of it's own... I'd like to redesign this quite a lot. A start is to split between "recoding" and "playback", respectively. We should also have a way to enable events for recording and playback independently.

The split between LootrunUtils and LootrunFeature also seems rather arbitrary. I think we should consider making selected parts of LootrunUtils and LootrunFeature into a LootrunManager. Maybe all of it. 

That change is worth a PR of it's own, so I won't say much more right now on this class.

Reply:
Got it, new PR later.